index.js

```diff
@@ -208,7 +208,7 @@ async function sendResetAlert(locationid) {
 
 // Heartbeat Helper Functions
 async function checkHeartbeat() {
-  const locations = await db.getLocations()
+  const locations = await db.getActiveLocations()
```

Comment:
No need to check heartbeat for inactive Locations. In fact, that something is known-down might be the reason why we set a Location to inactive for a period of time.